docs/config/lua/config/command_palette_rows.md

```diff
@@ -0,0 +1,11 @@
+---
+tags:
+  - appearance
+  - command_palette
+---
+# `command_palette_rows = 14`
+
```

Comment:
Please add version information so that we can tell when this is released:

```suggestion
# `command_palette_rows = 14`

{{since('nightly')}}
```